### PR TITLE
Fix java min benchmarking.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
As discussed in #969, java CI should run minimal benchmark on every PR.
Features:
* Fixes for benchmarking app for #658
* Add minimal benchmark run in CI (was on demand only)
* Fix java benchmaking app to comply with others (`--option` instead of `-option`) and corresponding script update

```
$ ./gradlew :benchmarks:run --args="--help"

> Task :client:buildRustRelease
    Finished release [optimized + debuginfo] target(s) in 5.01s

> Task :benchmarks:run
usage: glide
    --clientCount <arg>       Number of clients to run [1]
    --clients <arg>           one of: all|jedis|lettuce|glide
    --clusterModeEnabled      Is cluster-mode enabled, other standalone
                              mode is used [false]
    --concurrentTasks <arg>   Number of concurrent tasks [100, 1000]
    --configuration <arg>     Configuration flag [Release]
    --dataSize <arg>          Data block size [100 4000]
    --debugLogging            Verbose logs [false]
 -h,--help                    Print this message
    --host <arg>              Hostname [localhost]
    --minimal                 Run benchmark in minimal mode
    --port <arg>              Port number [6379]
    --resultsFile <arg>       Result filepath (stdout if empty) []
    --tls                     TLS [false]
```

### CODE REVIEW

**Review by acarbonetto — APPROVED** [review]
Can you also clean up the dashes in the README:
https://github.com/Bit-Quill/glide-for-redis/blob/bd0bd9bf7c265b1fe1060527da1d1d29142ce7c7/java/README.md?plain=1#L109-L125

**Review by acarbonetto — COMMENTED** [review]
Any reason why python/node jobs are failing?

**Comment by Yury-Fridlyand** [comment]
They are failing on our repo only. We need to investigate this.
Thank you for reminding about readme. Updated in 7103d596.

File: java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java [hunk 1]
```diff
@@ -66,36 +66,66 @@ private static Options getOptions() {
         // create the Options
         Options options = new Options();
 
-        options.addOption(Option.builder("help").desc("print this message").build());
+        options.addOption(Option.builder("h").longOpt("help").desc("Print this message").build());
```
Comment (jonathanl-bq) [inline java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java:69]: Are we going to update this message later? Or is it meant to just be a dummy message? If we want to update this later, let's make sure that it doesn't fall through the cracks.

**Review by jonathanl-bq — APPROVED** [review]
left one comment, but otherwise seems fine to me

File: java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java (hunk 1 above)
Comment (Yury-Fridlyand) [reply java/benchmarks/src/main/java/glide/benchmarks/BenchmarkingApp.java:69]: A dummy message

**Pull request merged**